README.rst

```diff
@@ -108,7 +110,7 @@ If you want to make an RPC and continue an existing trace, you can inject the cu
         current_span = tracer.get_span(request)
         text_carrier = {}
         opentracing_tracer.inject(span, opentracing.Format.TEXT_MAP, text_carrier)
-        for k, v in text_carrier.iteritems():
+        for k, v in text_carrier.items():
```

Comment:
This makes for a python 2/3 compatible example. I could have gone the route of using `six` I suppose.

Reply:
That is fine - it's only the README, so nobody gets hurt here ;)